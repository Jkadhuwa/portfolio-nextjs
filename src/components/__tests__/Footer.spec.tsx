import React from "react";
import "@testing-library/jest-dom";
import { render, screen, fireEvent, cleanup } from "@testing-library/react";
import FontAwesomeIcon from "../../__mocks__/@fortawesome/fontAwesomeIcon";

import Footer from "@components/layout/Footer";

afterEach(cleanup);
beforeAll(() => FontAwesomeIcon());
jest.mock("../../public/logo.png", () => {
  return {
    default: { src: "/public/logo.png", height: 100, width: 100 },
  };
});

const createWrapper = () => {
  return render(<Footer />);
};
describe("Test ExpertiseCard component", () => {
  describe("Snapshot Tests", () => {
    const wrapper = createWrapper();

    expect(wrapper).toMatchSnapshot();
  });

  describe("Footer Unit Tests", () => {
    it("Should render intro correctly", () => {
      createWrapper();
    });

    it("should have correct heading", () => {
      createWrapper();
      expect(screen.getByRole("heading")).toHaveTextContent(
        "Let's Collaborate!"
      );
    });
    it("should have correct heading", () => {
      createWrapper();
      expect(screen.getByTestId("dp")).toHaveAttribute(
        "src",
        "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      );
    });
  });
});
