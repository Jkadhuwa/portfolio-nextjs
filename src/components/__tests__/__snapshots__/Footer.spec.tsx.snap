// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col bg-slate-900 md:mx-12 py-12"
      >
        <section
          class="md:grid md:grid-cols-2 gap-6"
        >
          <div
            class="px-8 lg:px-32"
          >
            <div
              class="flex justify-center h-20 w-20 rounded-full bg-gray-300 mb-12"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27100%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt="profile_pic"
                  class="rounded-full "
                  data-nimg="intrinsic"
                  data-testid="dp"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="grid mb-12"
            >
              <span
                class="text-gray-200 text-xl font-medium"
              >
                Musinda Kadhuwa
              </span>
              <span
                class="text-slate-400 font-light text-sm"
              >
                Software Engineer
              </span>
            </div>
            <div
              class="grid"
            >
              <span
                class="text-slate-400 font-light text-sm"
              >
                Email me at:
              </span>
              <span
                class="text-gray-200 text-xl font-medium"
              >
                justinemsinda@gmail.com
              </span>
            </div>
          </div>
          <div
            class="px-8 lg:mt-12 "
          >
            <div
              class="text-gray-200 text-4xl font-extrabold"
            >
              <h1>
                Let's Collaborate!
              </h1>
            </div>
            <div
              class="flex flex-col text-gray-400 text-sm"
            >
              <p
                class="leading-8"
              >
                I am open to discussing personal and opensource projects, Let's create amazing products together
              </p>
              <div
                class="flex space-x-4 mt-12"
              >
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="/contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-dribbble w-4 cursor-pointer"
                      data-icon="dribbble"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M256 8C119.252 8 8 119.252 8 256s111.252 248 248 248 248-111.252 248-248S392.748 8 256 8zm163.97 114.366c29.503 36.046 47.369 81.957 47.835 131.955-6.984-1.477-77.018-15.682-147.502-6.818-5.752-14.041-11.181-26.393-18.617-41.614 78.321-31.977 113.818-77.482 118.284-83.523zM396.421 97.87c-3.81 5.427-35.697 48.286-111.021 76.519-34.712-63.776-73.185-116.168-79.04-124.008 67.176-16.193 137.966 1.27 190.061 47.489zm-230.48-33.25c5.585 7.659 43.438 60.116 78.537 122.509-99.087 26.313-186.36 25.934-195.834 25.809C62.38 147.205 106.678 92.573 165.941 64.62zM44.17 256.323c0-2.166.043-4.322.108-6.473 9.268.19 111.92 1.513 217.706-30.146 6.064 11.868 11.857 23.915 17.174 35.949-76.599 21.575-146.194 83.527-180.531 142.306C64.794 360.405 44.17 310.73 44.17 256.323zm81.807 167.113c22.127-45.233 82.178-103.622 167.579-132.756 29.74 77.283 42.039 142.053 45.189 160.638-68.112 29.013-150.015 21.053-212.768-27.882zm248.38 8.489c-2.171-12.886-13.446-74.897-41.152-151.033 66.38-10.626 124.7 6.768 131.947 9.055-9.442 58.941-43.273 109.844-90.795 141.978z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="https://www.linkedin.com/in/kadhuwa-musinda-a39346123/"
                    rel="noopener noreferrer"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-linkedin w-4 cursor-pointer"
                      data-icon="linkedin"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 448 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="/contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-github w-4 cursor-pointer"
                      data-icon="github"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 496 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="/contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-bitbucket w-4 cursor-pointer"
                      data-icon="bitbucket"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M22.2 32A16 16 0 0 0 6 47.8a26.35 26.35 0 0 0 .2 2.8l67.9 412.1a21.77 21.77 0 0 0 21.3 18.2h325.7a16 16 0 0 0 16-13.4L505 50.7a16 16 0 0 0-13.2-18.3 24.58 24.58 0 0 0-2.8-.2L22.2 32zm285.9 297.8h-104l-28.1-147h157.3l-25.2 147z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="/contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-medium-m w-4 cursor-pointer "
                      data-icon="medium-m"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M71.5 142.3c.6-5.9-1.7-11.8-6.1-15.8L20.3 72.1V64h140.2l108.4 237.7L364.2 64h133.7v8.1l-38.6 37c-3.3 2.5-5 6.7-4.3 10.8v272c-.7 4.1 1 8.3 4.3 10.8l37.7 37v8.1H307.3v-8.1l39.1-37.9c3.8-3.8 3.8-5 3.8-10.8V171.2L241.5 447.1h-14.7L100.4 171.2v184.9c-1.1 7.8 1.5 15.6 7 21.2l50.8 61.6v8.1h-144v-8L65 377.3c5.4-5.6 7.9-13.5 6.5-21.2V142.3z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
                <span
                  class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
                >
                  <a
                    href="/contact"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-twitter w-4 cursor-pointer "
                      data-icon="twitter"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </span>
              </div>
            </div>
          </div>
        </section>
        <span
          class="bg-gray-800 h-0.5 mt-12"
        />
        <section
          class="flex flex-col space-y-4 md:space-y-0 items-center md:flex-row md:justify-between text-xs mt-12 md:mx-8 lg:mx-32"
        >
          <div
            class="space-x-4 text-gray-300"
          >
            <span>
              <a
                href="/"
              >
                About
              </a>
            </span>
            <span>
              <a
                href="/portfolio"
              >
                Works
              </a>
            </span>
            <span>
              <a
                href="/resume"
              >
                Resume
              </a>
            </span>
            <span>
              <a
                href="/contact"
              >
                Contact
              </a>
            </span>
          </div>
          <div
            class="text-gray-600"
          >
            ©
            2022
             Musinda Kadhuwa - (Uninet, inc)
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col bg-slate-900 md:mx-12 py-12"
    >
      <section
        class="md:grid md:grid-cols-2 gap-6"
      >
        <div
          class="px-8 lg:px-32"
        >
          <div
            class="flex justify-center h-20 w-20 rounded-full bg-gray-300 mb-12"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27100%27%20height=%27100%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="profile_pic"
                class="rounded-full "
                data-nimg="intrinsic"
                data-testid="dp"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="grid mb-12"
          >
            <span
              class="text-gray-200 text-xl font-medium"
            >
              Musinda Kadhuwa
            </span>
            <span
              class="text-slate-400 font-light text-sm"
            >
              Software Engineer
            </span>
          </div>
          <div
            class="grid"
          >
            <span
              class="text-slate-400 font-light text-sm"
            >
              Email me at:
            </span>
            <span
              class="text-gray-200 text-xl font-medium"
            >
              justinemsinda@gmail.com
            </span>
          </div>
        </div>
        <div
          class="px-8 lg:mt-12 "
        >
          <div
            class="text-gray-200 text-4xl font-extrabold"
          >
            <h1>
              Let's Collaborate!
            </h1>
          </div>
          <div
            class="flex flex-col text-gray-400 text-sm"
          >
            <p
              class="leading-8"
            >
              I am open to discussing personal and opensource projects, Let's create amazing products together
            </p>
            <div
              class="flex space-x-4 mt-12"
            >
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="/contact"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-dribbble w-4 cursor-pointer"
                    data-icon="dribbble"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.252 8 8 119.252 8 256s111.252 248 248 248 248-111.252 248-248S392.748 8 256 8zm163.97 114.366c29.503 36.046 47.369 81.957 47.835 131.955-6.984-1.477-77.018-15.682-147.502-6.818-5.752-14.041-11.181-26.393-18.617-41.614 78.321-31.977 113.818-77.482 118.284-83.523zM396.421 97.87c-3.81 5.427-35.697 48.286-111.021 76.519-34.712-63.776-73.185-116.168-79.04-124.008 67.176-16.193 137.966 1.27 190.061 47.489zm-230.48-33.25c5.585 7.659 43.438 60.116 78.537 122.509-99.087 26.313-186.36 25.934-195.834 25.809C62.38 147.205 106.678 92.573 165.941 64.62zM44.17 256.323c0-2.166.043-4.322.108-6.473 9.268.19 111.92 1.513 217.706-30.146 6.064 11.868 11.857 23.915 17.174 35.949-76.599 21.575-146.194 83.527-180.531 142.306C64.794 360.405 44.17 310.73 44.17 256.323zm81.807 167.113c22.127-45.233 82.178-103.622 167.579-132.756 29.74 77.283 42.039 142.053 45.189 160.638-68.112 29.013-150.015 21.053-212.768-27.882zm248.38 8.489c-2.171-12.886-13.446-74.897-41.152-151.033 66.38-10.626 124.7 6.768 131.947 9.055-9.442 58.941-43.273 109.844-90.795 141.978z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="https://www.linkedin.com/in/kadhuwa-musinda-a39346123/"
                  rel="noopener noreferrer"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-linkedin w-4 cursor-pointer"
                    data-icon="linkedin"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="/contact"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-github w-4 cursor-pointer"
                    data-icon="github"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 496 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="/contact"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-bitbucket w-4 cursor-pointer"
                    data-icon="bitbucket"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M22.2 32A16 16 0 0 0 6 47.8a26.35 26.35 0 0 0 .2 2.8l67.9 412.1a21.77 21.77 0 0 0 21.3 18.2h325.7a16 16 0 0 0 16-13.4L505 50.7a16 16 0 0 0-13.2-18.3 24.58 24.58 0 0 0-2.8-.2L22.2 32zm285.9 297.8h-104l-28.1-147h157.3l-25.2 147z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="/contact"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-medium-m w-4 cursor-pointer "
                    data-icon="medium-m"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M71.5 142.3c.6-5.9-1.7-11.8-6.1-15.8L20.3 72.1V64h140.2l108.4 237.7L364.2 64h133.7v8.1l-38.6 37c-3.3 2.5-5 6.7-4.3 10.8v272c-.7 4.1 1 8.3 4.3 10.8l37.7 37v8.1H307.3v-8.1l39.1-37.9c3.8-3.8 3.8-5 3.8-10.8V171.2L241.5 447.1h-14.7L100.4 171.2v184.9c-1.1 7.8 1.5 15.6 7 21.2l50.8 61.6v8.1h-144v-8L65 377.3c5.4-5.6 7.9-13.5 6.5-21.2V142.3z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
              <span
                class="flex items-center justify-center hover:bg-gray-600 h-8 w-8 rounded-full"
              >
                <a
                  href="/contact"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-twitter w-4 cursor-pointer "
                    data-icon="twitter"
                    data-prefix="fab"
                    focusable="false"
                    role="img"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                      fill="currentColor"
                    />
                  </svg>
                </a>
              </span>
            </div>
          </div>
        </div>
      </section>
      <span
        class="bg-gray-800 h-0.5 mt-12"
      />
      <section
        class="flex flex-col space-y-4 md:space-y-0 items-center md:flex-row md:justify-between text-xs mt-12 md:mx-8 lg:mx-32"
      >
        <div
          class="space-x-4 text-gray-300"
        >
          <span>
            <a
              href="/"
            >
              About
            </a>
          </span>
          <span>
            <a
              href="/portfolio"
            >
              Works
            </a>
          </span>
          <span>
            <a
              href="/resume"
            >
              Resume
            </a>
          </span>
          <span>
            <a
              href="/contact"
            >
              Contact
            </a>
          </span>
        </div>
        <div
          class="text-gray-600"
        >
          ©
          2022
           Musinda Kadhuwa - (Uninet, inc)
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
